.config/api-client-configs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "bf2fc6cc711aee1a0c2a/kas-fleet-manager/kas-fleet-manager.yaml": {
+  "kas-fleet-manager.yaml": {
```

Comment:
This is fine, but what happens when another API repo just has `openapi.yaml`? Combining them makes it unique.